virttest/utils_test/qemu/__init__.py

```diff
@@ -585,12 +585,7 @@ def update_vm_after_hotplug(self, vm, dev):
         """
         error_context.context("Update VM object after hotplug memory")
         dev_type, name = dev.get_qid().split('-')
-        if isinstance(dev, qdevices.Memory):
-            backend = self.params.object_params(name).get("backend_mem",
-                                                          "memory-backend-ram")
-            attrs = dev.__attributes__[backend][:]
-        else:
-            attrs = dev.__attributes__[:]
+        attrs = dev.attrs
```

Comment:
Shall we consider to add a default value for `backend_mem`?
I am afraid that it will hit regression when calling `update_vm_after_hotplug` without `backend_mem = memory-backend-ram`.

Reply:
The default value is already set in memory_object_define_by_params().  And no regression is found when run memory hotplug cases.

Reply:
Okay, got it.